### DISCUSSION TITLE
About logging off rdp sessions

### DISCUSSION THREAD

**Discussion by GG668** [FreeRDP/FreeRDP#8923] [Q&A]
Does freerdp have a function to log out of a session? Make the session disappear completely instead of being in a disconnected state

**Comment by akallabeth** [comment]
that is a RDP server feature, e.g. you need to log out on your server while in a session.

**Reply by GG668** [reply]
That means we have no way to log out of the session via the rdp protocol, we have to click on logout manually, right?

**Reply by akallabeth** [reply]
there is none I´m aware of, so yes, you need to log out manually or configure the server to do so when a session is disconnected.

**Reply by GG668** [reply]
For example, logging out of a session via Task Manager

**Reply by GG668** [reply]
Thank you
